src/App.tsx

```diff
@@ -1,10 +1,59 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos, postTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorType, HandleErrors } from './components/HandleErrors/HandleErros';
+import { TodoList } from './components/TodoList/TodoList';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { Filter } from './types/Filter';
+import { Loader } from './components/Loader/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setError] = useState<ErrorType | null>(null);
+  const [filter, setFilter] = useState<Filter>('all');
+  const [loader, setLoader] = useState(false);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setError('load');
+        setTimeout(() => setError('load'), 3000);
+      });
+  }, []);
+
+  const handleCloseError = () => {
```

Comment:
always use useCallback on handlers and timers to improve performance -> https://semaphoreci.com/blog/react-usecallback-hook#:~:text=useCallback%20is%20most%20commonly%20used,as%20event%20handlers%20and%20timers.

Reply:
Hey João, in this part of the task it is not yet necessary to exclude todos.